### ISSUE TITLE
My Projects button not in tab order

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2333]
**Describe the bug**
My projects button that goes to project view not in tab order / cannot be accessed

**To Reproduce**
1. Go to home screen
2. try to tab to ``My Projects`` button above ``New Project``
3. See error

**Expected behavior**

should be able to open ``My Projects`` by tabbing to it

**Screenshots**

<img width="290" alt="Screen Shot 2019-08-02 at 2 45 49 PM" src="https://user-images.githubusercontent.com/5615930/62400559-7ce7be80-b534-11e9-886b-d16f8ac45761.png">
